src/Frontend-GNOME/Views/MessageTextView.cs

```diff
@@ -51,6 +52,7 @@ public class MessageTextView : Gtk.TextView
         private Gdk.Color    _MarkerlineColor = new Gdk.Color(255, 0, 0);
         private int          _MarkerlineBufferPosition;
         private int          _BufferLines = -1;
+        private IconCache    _EmojiCache = new IconCache("emoji");
```

Comment:
It is private, so we would only use it inside this class. If we wanted to make it public, we'd be changing anyway, no?